app/posts/2021-04-11-ab-test-inline-create-account-form.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'A/B test: an inline form to encourage job alert creators to create an account'
+description: An inline form encouraged more job alert creators to create an account.
+date: 2021-03-04
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+When jobseekers create a job alert, they are not required to create an account. This is so that we don’t discourage them from creating an alert by placing an unnecessary extra step in the journey. However, if they do create an account, then they have access to a greater range of functionality - better management of job alerts, and also the ability to save and apply for jobs. So, we wanted to encourage as many job alert creators as possible to create an account without requiring them to do so.
+
+## Hypothesis
+
+If we allow job alert creators to create accounts from a form within the job alert confirmation screen itself, instead of just linking to it then more of them will create an account.
```

Comment:
```suggestion
If we allow job alert creators to create accounts from a form within the job alert confirmation screen itself, instead of just linking to it, then more of them will create an account.
```